### PR TITLE
Test that maps are below max size

### PR DESCRIPTION
**Pull request by amosr**
This should be failing consistently but isn't. The generated programs aren't interesting enough.
/jury approved @jystic

### CODE REVIEW

**Comment by jystic** [comment]
would one of the corpus programs fail consistently?

**Comment by amosr** [comment]
Yeah, good idea. I think it's worth having both. I'm going to leave this PR here until I've got the rest implemented

**Comment by amosr** [comment]
includes corpus tests for max map size

! @jystic @tranma

**Comment by jystic** [comment]
👍

File: icicle-compiler/test/test.hs [hunk 1]
```diff
@@ -80,13 +77,12 @@ main
 
         , Icicle.Test.Foreign.Array.tests
 
-        -- , Icicle.Test.Dictionary.Types.tests
-
```
Comment (jystic) [inline icicle-compiler/test/test.hs]: 🍾

**Pull request merged**